__mocks__/react-native-safe-area-context.tsx

```diff
@@ -0,0 +1,61 @@
+import type {ForwardedRef, ReactNode} from 'react';
+import React, {forwardRef} from 'react';
+import {View} from 'react-native';
+import type {EdgeInsets, useSafeAreaFrame as LibUseSafeAreaFrame, WithSafeAreaInsetsProps} from 'react-native-safe-area-context';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+
+type SafeAreaProviderProps = ChildrenProps;
+type SafeAreaConsumerProps = {
+    children?: (insets: EdgeInsets) => ReactNode;
+};
+type SafeAreaInsetsContextValue = {
+    Consumer: (props: SafeAreaConsumerProps) => ReactNode;
+};
+
+const insets: EdgeInsets = {
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+};
+
+function withSafeAreaInsets(WrappedComponent: React.ComponentType<WithSafeAreaInsetsProps & {ref: ForwardedRef<unknown>}>) {
+    function WithSafeAreaInsets(props: WithSafeAreaInsetsProps & {forwardedRef: React.ForwardedRef<unknown>}) {
+        return (
+            <WrappedComponent
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...props}
+                // eslint-disable-next-line react/prop-types
+                ref={props.forwardedRef}
+                insets={insets}
+            />
+        );
+    }
+
+    const WithSafeAreaInsetsWithRef = forwardRef((props: WithSafeAreaInsetsProps, ref: ForwardedRef<unknown>) => (
+        <WithSafeAreaInsets
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            forwardedRef={ref}
+        />
+    ));
+
+    return WithSafeAreaInsetsWithRef;
+}
+
+const SafeAreaView: typeof View = View;
```

Comment:
```suggestion
const SafeAreaView = View;
```

Reply:
@kubabutkiewicz are you going to commit this suggestion?

Reply:
^ this is my only concern @kubabutkiewicz

Reply:
its commited, I missed that comment